src/audio/copier/copier.c

```diff
@@ -1387,7 +1387,7 @@ static int do_endpoint_copy(struct comp_dev *dev)
 			break;
 		case SOF_COMP_DAI:
 			if (cd->endpoint_num == 1)
```

Comment:
BTW if (cd->endpoint_num == 1) should be remove here since it is not a multiple-endpoint case, or it will confuse reader that what is else ?